content/docs/command-reference/remote/add.md

```diff
@@ -171,18 +171,15 @@ $ dvc remote modify --local myremote connection_string 'mysecret'
 > `--local` option, so it's written to a Git-ignored config file. See
 > `dvc remote modify` for a full list of Azure parameters.
 
-The Azure Blob Storage remote can also be configured globally via environment
-variables:
+This remote type can also be configured via environment variables, for example:
 
 ```dvc
 $ export AZURE_STORAGE_CONNECTION_STRING='mysecret'
 $ dvc remote add -d myremote azure://mycontainer/path
 ```
 
-> 💡 Make sure the value is quoted so its processed correctly by the console.
-
-See `dvc remote modify` for a full list of Azure parameters and environment
-variables.
+See `dvc remote modify` for a full list of Azure parameters and env vars for
```

Comment:
I might be missing context on this. How does `dvc remote modify` shows options? 
```
 $ dvc remote modify origin       
ERROR: the following arguments are required: option
usage: dvc remote modify [-h] [--global | --system | --project | --local] [-q | -v] [-u] name option [value]

Modify the configuration of a data remote.
Documentation: <https://man.dvc.org/remote/modify>

positional arguments:
  name           Name of the remote
  option         Name of the option to modify.
  value          (optional) Value of the option.

optional arguments:
  -h, --help     show this help message and exit
  --global       Use global config.
  --system       Use system config.
  --project      Use project config (.dvc/config).
  --local        Use local config (.dvc/config.local).
  -q, --quiet    Be quiet.
  -v, --verbose  Be verbose.
  -u, --unset    Unset option.

```

Reply:
Yeah those code blocks become links in the cmd ref:

![image](https://user-images.githubusercontent.com/1477535/115765111-f7d9b400-a36b-11eb-9a31-f1dd1aa17dea.png)

So you click on it and it goes to the `remote modify` ref, where you can find the Azure section with all the auth details. That's the idea.

Reply:
Ah, TIL.

Reply:
Haha. So approved? 🙂